perf/Makefile

```diff
@@ -4,10 +4,12 @@
 #
 .PHONY: run
 run:
+	docker pull returntocorp/semgrep:develop
```

Comment:
but we also want to run bencmark with the local semgrep in your PATH no?
using the container is also slower.

Reply:
For using the local `semgrep`, you just run the script `./run-benchmarks` without options.

Reply:
I added clarifications to readme and makefile.